src/neo4j_genai/indexes.py

```diff
@@ -278,13 +278,13 @@ def upsert_vector(
         Neo4jInsertionError: If upserting of the vector fails.
     """
     try:
-        query = """
-        MATCH (n)
-        WHERE elementId(n) = $id
-        WITH n
-        CALL db.create.setNodeVectorProperty(n, $embedding_property, $vector)
-        RETURN n
-        """
+        query = (
+            "MATCH (n)"
```

Comment:
Why change the multiline string here?

Reply:
If you use `"""` to create multiline strings any indentation in your code will be included in the string. This means when you write your tests you need to make sure the indentation in the multi-line strings in the tests matches the indentation in the parts of the code they're testing, which can make the multi-line strings in tests look a bit weird. If you instead write multi-line strings this way you don't have to worry about it

Reply:
Have you tested these queries? I think you may need to add a space to the end of each line this way.

Regardless, I think using """ makes it more readable though, we had an issue previously where with the proposed way, someone missed adding a space at the end as it was not very readable. I proposed switching back to using """ but for tests use something like https://docs.python.org/3/library/textwrap.html . What do you think?

Reply:
I'm not sure how that'd work for tests such as `test_upsert_node_with_embedding`. You'd need to call textwrap within `upsert_vector` and the test to make sure the strings match. Maybe the way around this is to just move the queries to the `neo4j_queries.py` file and import them in both tests and the upsert functions.

Reply:
I think if it works, maybe there's no need to overcomplicate it for now. I'd be happy with the original approach provided the query runs